test/sma.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env mocha */
+'use strict'
+
+const assert = require('assert')
+const SMA = require('../lib/sma')
+
+const smaPeriod = [5]
+const candles = [
+  { open: 6373, high: 6375.9, low: 6369.2, close: 6374.5 },
+  { open: 6408.8, high: 6408.9, low: 6366, close: 6372.9 },
+  { open: 6390.1, high: 6400, low: 6376, close: 6380 },
+  { open: 6380, high: 6380, low: 6380, close: 6380 },
+  { open: 6480, high: 6490.1, low: 6375, close: 6382.1 },
+  { open: 6382.1, high: 6418.9, low: 6377, close: 6390 },
+]
+
```

Comment:
what should happen when we set the period to 5, but just add 3 values?

should it return an error?

Reply:
talib seems to return an empty result:

```
{ begIndex: 0, nbElement: 0, result: { outReal: [] } }
```

Reply:
maybe we should return `null`?

Reply:
yeahh .. just checked .. we have a buffer array.. unless the buffer length is >= period should not add in the values .. so that the result is returned as 0 (if it's fine) or return null